manifest.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "b2b-organizations",
   "vendor": "vtex",
-  "version": "1.25.0",
+  "version": "1.26.0",
```

Comment:
Can you revert this line changing the version. This should only be changed by the IO bot, after merging.